person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
- [OPTIONAL] For better code organization, it will be nice to move this `getter and setter` methods to the top of the class.

Reply:
Ok, bro.
README updated. 👍